tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/provider/TieredStorageConfiguration.java

```diff
@@ -63,6 +63,10 @@
     public static final String METADATA_FIELD_MAX_BLOCK_SIZE = "maxBlockSizeInBytes";
     public static final String METADATA_FIELD_READ_BUFFER_SIZE = "readBufferSizeInBytes";
     public static final String OFFLOADER_PROPERTY_PREFIX = "managedLedgerOffload";
+    public static final String MAX_SEGMENT_TIME_IN_SECOND = "maxSegmentTimeInSecond";
+    public static final long DEFAULT_MAX_SEGMENT_TIME_IN_SECOND = 600;
+    public static final String MAX_SEGMENT_SIZE_IN_BYTES = "maxSegmentSizeInBytes";
+    public static final long DEFAULT_MAX_SEGMENT_SIZE_IN_BYTES = 1024 * 1024 * 1024;
```

Comment:
We also need to add the minOffloadSegmentRolloverTimeInSeconds to avoid the segments rollover too often.

Reply:
Do you mean segment may become larger than `MAX_SEGMENT_SIZE_IN_BYTES` if the time it reaches `MAX_SEGMENT_SIZE_IN_BYTES` is shorter than `minOffloadSegmentRolloverTimeInSeconds`? If so, maybe we need a `REAL_MAX_SEGMENT_SIZE_IN_BYTES` to avoid the segment become too big.

Reply:
`minOffloadSegmentRolloverTimeInSeconds` related implementations added